modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -135,6 +135,20 @@ def query_params(url)
       expect(json['data'].length).to eq(10)
     end
 
+    it 'responds to GET #index with state code' do
+      setup_pdx
+      get base_query_path, params: 'state=WA', headers: accept_json
+      expect(response).to be_success
+      expect(response.body).to be_a(String)
+      json = JSON.parse(response.body)
+      expect(json['data'].length).to eq(2)
+    end
+
+    it 'responds to invalid states on GET #index with a 4xx status code' do
```

Comment:
Total nitpick, but the rest of the error codes are grouped together under `context 'with invalid request parameters' do`.

Reply:
Not nitpicky. Moving it!